src/Service.Tests/SqlTests/RestApiTests/Delete/DwSqlDeleteApiTest.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.SqlTests.RestApiTests.Delete
+{
+    /// <summary>
+    /// Test REST Apis validating expected results are obtained.
+    /// </summary>
+    [TestClass, TestCategory(TestCategory.DWSQL)]
+    public class DwSqlDeleteApiTests : DeleteApiTestBase
+    {
+        private static Dictionary<string, string> _queryMap = new()
+        {
+            {
+                "DeleteOneWithStoredProcedureTest",
+                $"SELECT [id] FROM { _integrationTableName } " +
+                $"WHERE id = 14"
+            }
+        };
+        #region Test Fixture Setup
+
+        /// <summary>
+        /// Sets up test fixture for class, only to be run once per test run, as defined by
+        /// MSTest decorator.
+        /// </summary>
+        /// <param name="context"></param>
```

Comment:
nit: if the param doesnt need a description it probably doesnt need to be listed since we can see it in the method sig.